bridge/opentracing/bridge_test.go

```diff
@@ -574,3 +574,89 @@ func TestBridgeSpanContextPromotedMethods(t *testing.T) {
 		assert.True(t, spanContext.(spanContextProvider).HasTraceID())
 	})
 }
+
+func TestBridgeCarrierBaggagePropagation(t *testing.T) {
+	testCases := []struct {
+		name             string
+		format           ot.BuiltinFormat
+		carrier          interface{}
+		baggageItems     []bipBaggage
+		wantBaggageItems []bipBaggage
+	}{
+		{
+			name:    "HTTPHeadersCarrier",
+			format:  ot.HTTPHeaders,
+			carrier: ot.HTTPHeadersCarrier(http.Header{}),
+			baggageItems: []bipBaggage{
+				{
+					key:   "foo",
+					value: "bar",
+				},
+			},
+			wantBaggageItems: []bipBaggage{
+				{
+					key:   "Foo",
```

Comment:
why is this capitalized? If it is to demonstrate that case doesn't matter, they I would do use a separate test case for that that is explicit about it.

Reply:
The capitalization isn't important to the test, it seems `setBaggageItem` would set the `CanonicalHeaderKey` to capitalized
https://github.com/open-telemetry/opentelemetry-go/blob/885210bf33238f8c6cc94c3f3711b0036fbe77b1/bridge/opentracing/bridge.go#L76-L78

Reply:
> would set it to capitalized through

which is another bug. Baggage keys are case sensitive, they are not supposed to be mangled this way -- https://github.com/w3c/baggage/blob/ebda53c1d33031d95f1104f44840d3d4b875eb89/baggage/HTTP_HEADER_FORMAT_RATIONALE.md?plain=1#L55